src/app/api/gql/scheduling-dataset/root-schedule.graphql

```diff
@@ -0,0 +1,22 @@
+query getDatasetIngestSchedule($datasetId: DatasetID!) {
+  datasets {
+    byId(datasetId: $datasetId) {
+      flows {
+        configs {
+          __typename
+          byType(datasetFlowType: INGEST) {
```

Comment:
Not enough abstraction:
 - make flow type a parameter, there might be other flow types very soon that will also have a schedule
 - the name of the query should not mention "ingest" as well
 - the file name should not include "root", as there might be other flow types where there is no such restriction whether the dataset is root or derivative, i.e. compaction will be possible by schedule for any kind of dataset

Reply:
Fixed